login-workflow/package.json

```diff
@@ -32,17 +32,17 @@
         "i18next-browser-languagedetector": "^8.0.0"
     },
     "peerDependencies": {
-        "@brightlayer-ui/react-components": "^6.1.2",
+        "@brightlayer-ui/react-components": "7.0.0",
         "@emotion/react": "^11.7.1",
         "@emotion/styled": "^11.6.0",
-        "@mui/icons-material": "^5.10.15",
-        "@mui/material": "^5.10.15",
+        "@mui/icons-material": "^6.1.4",
+        "@mui/material": "^6.1.4",
```

Comment:
New peer dependencies means this needs to be a new major version of the workflow, not a minor

Reply:
Changed to major version v6